src/_tutorials/dart-vm/httpserver.md

```diff
@@ -154,21 +152,22 @@ listening, and responding.
 
 ## Binding a server to a host and port {#binding}
 
-_Example for this section: hello_world_server.dart._
+_Example for this section:_
+[hello_world_server.dart.]({{gh-path}}/bin/hello_world_server.dart)
 
 The first line of code in `main()` uses `HttpServer.bind()` to create an
 [HttpServer][] object and bind it to a host and port.
 
+<?code-excerpt "httpserver/bin/hello_world_server.dart (bind)"?>
 {% prettify dart %}
-var [[highlight]]requestServer[[/highlight]] =
-    await HttpServer.bind([[highlight]]InternetAddress.LOOPBACK_IP_V4, 4040[[/highlight]]);
-...
+var server = await HttpServer.bind(
+  InternetAddress.LOOPBACK_IP_V4,
+  4040,
+);
 {% endprettify %}
 <div class="prettify-filename">hello_world_server.dart</div>
 
 The code uses `await` to call the `bind` method asynchronously.
-When the bind is successful, the new HttpServer object is assigned
```

Comment:
The name of the declared variable changed and so that line of prose required editing. I decided to just drop that sentence because IMHO: (1) The code self explanatory. (2) This is especially the case given that the paragraph immediately before the code snippet reads:
> The first statement in `main()` uses `HttpServer.bind()` to create an [HttpServer][] object and bind it to a host and port.

(Otherwise, the statement is essentially describing how a variable declaration with an initializer works.)